EIPS/eip-6789.md

```diff
@@ -0,0 +1,88 @@
+---
+eip: 6789
+title: Rename `gas` to `mana`
+description: This EIP picks up on Vitalik Buterin's (@vbuterin) original proposal in 2015 to rename `gas` to `mana`
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/eip-6789-rename-gas-to-mana/13570
+status: Draft
+type: Standards Track
+category: Interface
```

Comment:
```suggestion
type: Standards Track
category: Core
```
I was trying to determine whether to have it be Meta, Informational, or Core. I think Core most accurately describes this EIP, but I encourage other editors to chime in.

Reply:
Here we go again :). That was also my first choice, but Matt had another opinion here https://github.com/ethereum/EIPs/pull/6789#discussion_r1149559967. It seems my EIPs are always a challenge to assign a category. So, how should we proceed here?

Reply:
I would say that Interface is *definitely* not the right category for this.

Reply:
Fully agreed - I read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) again and it states:

> **Core**: [..] as well as changes that are not necessarily consensus critical but may be relevant to [“core dev” discussions](https://github.com/ethereum/pm) (for example, [EIP-90], [...]

=> This is definitely critical at some point to the "core dev" discussions.

Reply:
@Pandapip1 the precedence is clearly to use `Interface` for renaming mnemonics and the like. See [EIP-6](https://eips.ethereum.org/EIPS/eip-6).

Reply:
Okay, if there's a precedent, I'm okay with interface.

Reply:
Resolving this discussion for the moment.